resteasy-core/src/main/java/org/jboss/resteasy/util/BasicAuthHelper.java

```diff
@@ -13,7 +14,7 @@ public static String createHeader(String username, String password)
    {
       StringBuffer buf = new StringBuffer(username);
       buf.append(':').append(password);
-      return "Basic " + Base64.encodeBytes(buf.toString().getBytes(StandardCharsets.UTF_8));
+      return "Basic " + java.util.Base64.getEncoder().encodeToString(buf.toString().getBytes(StandardCharsets.UTF_8));
```

Comment:
there is no need for using FQDN because only one Base64 is imported now

Reply:
Fixed,thaks